.github/workflows/build.yml

```diff
@@ -0,0 +1,51 @@
+name: Build
+
+on: push
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x, 15.x]
```

Comment:
🙏 Node 16 was recently released, let's add that too.

Node 10 goes end-of-life and the end up April, I wonder if we should remove support? Maybe we just wait until dependencies start breaking.